force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -0,0 +1,112 @@
+import { LightningElement, api, track } from "lwc";
+import accountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+
+export default class EdaSettingsNavigation extends LightningElement {
+    labelReference = {
+        accountModelSettingsTitle,
+    };
+
+    @track settingsNavigationViewModel = {
+        navigationSections: [
+            {
+                label: "Setup Home",
+                page: "setuphome",
+                id: "setuphome",
+            },
+            {
+                label: "People and Groups",
+                id: "peopleAndGroups",
+                navigationSubSections: [
+                    {
+                        label: this.labelReference.accountModelSettingsTitle,
+                        page: "accountModelSettings",
+                        id: "accountModelSettings",
+                        menuItems: [
+                            { label: "Default Account Model", id: "defaultaccountmodel" },
+                            {
+                                label: "Administrative Account Record Type",
+                                menuitemkey: "administrativeAccountRecordType",
```

Comment:
should "menuitemkey" be replaced with "id" to align with the other menuItems?

Reply:
Good catch! Done